Sources/Swift/Extensions/StringExtensions.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+extension String {
+    /// Retrieve a character at a specific index in the string.
+    subscript (value: Int) -> Character {
```

Comment:
`m`: Codecov shows that we don't use all of these methods here. Either we remove them, or we keep them and we also add tests for these, please.